panoramapublic/src/org/labkey/panoramapublic/view/search/panoramaWebSearch.jsp

```diff
@@ -399,6 +399,7 @@
                                                 fieldLabel: 'Exact Matches Only',
                                                 name: exactProteinMatchesItemId,
                                                 itemId: exactProteinMatchesItemId,
+                                                id: exactProteinMatchesItemId,
```

Comment:
Shouldn't use hard-coded element IDs here.
WebParts are not guaranteed to be unique but HTML IDs should be.

Reply:
I did that change based on Ankur's suggestion, i will defer this to him @ankurjuneja

Reply:
@labkey-tchad this is an ext js element inside WebPart.

Reply:
@ankurjuneja I understand that but if you had two webparts, they would each have an ext js element with `id = "exactPeptideMatches-inputEl"`. Ext might not actually care about the ID but I've seen React behave strangely when two inputs have the same id.
Regardless, HTML ids are supposed to be unique.